src/pages/gwanyong/Login/Login.js

```diff
@@ -1,8 +1,175 @@
 import React from 'react';
+import { withRouter } from 'react-router';
+import Posting from './Posting/Posting';
+import './Login.scss';
+import '../../../styles/common.scss';
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: '',
+      pw: '',
+      btnColor: 'rgb(192, 223, 253)',
+      posting: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://10.58.6.223:8000/postings', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        this.setState({ posting: data.results });
+      });
+  }
+
+  handleSubmit = e => {
+    fetch('http://10.58.6.223:8000/postings', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        if (data.access_token) {
+          this.goToMain();
+        } else {
+          alert('id pw 확인');
+        }
+      });
+    e.preventDefault();
+  };
+
+  goToMain = () => {
+    let id = this.state.id;
+    let pw = this.state.pw;
```

Comment:
- 할당된 값이 변하는 경우가 아니라면 항상 const 로 선언해주세요!
- 구조분해 할당 문법을 사용하시면 좀 더 간결하게 코드를 작성하실 수 있을 것 같네요!
```suggestion
    const { id, pw } = this.state;
```